src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -804,14 +804,14 @@ def load_arguments(self, _):
     with self.argument_context('vm run-command show') as c:
         c.argument('vm_name', run_cmd_vm_name)
         c.argument('run_command_name', run_cmd_name_type)
-        c.argument('expand', help='The expand expression to apply on the operation.')
+        c.argument('status', action='store_true', help='Track the run command progress')
```

Comment:
It is recommended to use `--instance-view` instead of `--status`, because the semantics of `--instance-view` is clearer and can be consistent with the usage of other commands

Reply:
![Screenshot 2021-11-15 140722](https://user-images.githubusercontent.com/16612065/141731115-e0e30995-3174-41aa-ad52-df0ab8af0f42.png)
@PARADISSEEKR We have synced offline